test/features/deploy_quarkus_service.feature

```diff
@@ -154,3 +154,41 @@ Feature: Deploy quarkus service
     Examples:
       | native  | minutes |
       | enabled | 20      |
+
+  @usertasks
+  Scenario Outline: Deploy process-quarkus-example service to complete user tasks and native <native>
+    Given Kogito Operator is deployed
+    And Deploy quarkus example service "process-quarkus-example" with configuration:
+      | config | native      | <native> |
+      | config | persistence | disabled |
+    And Kogito application "process-quarkus-example" has 1 pods running within <minutes> minutes
+    And HTTP GET request on service "process-quarkus-example" with path "orders" is successful within 3 minutes
+
+    When HTTP POST request on service "process-quarkus-example" with path "orders" and body:
```

Comment:
`When Start "orders" process on service "process-quarkus-example" with body:`

Reply:
I prefer doing this as part of KOGITO-2001.

Reply:
Done